scripts/verify-phase-52-1-cli-command-contract.sh

```
@@ -0,0 +1,169 @@
+}
+
+for claim in "${forbidden_claims[@]}"; do
+  if contains_forbidden_outside_forbidden_section "${claim}"; then
+    echo "Forbidden Phase 52.1 CLI command contract claim: ${claim}" >&2
+    exit 1
+  fi
+done
+
+mac_user_home="$(printf '/%s/' 'Users')"
+unix_user_home="$(printf '/%s/' 'home')"
+windows_user_profile_backslash="$(printf '[A-Za-z]:\\\\%s\\\\' 'Users')"
```

Shellcheck (0.11.0)

[info] Want to escape a single quote? echo 'This is how it'\''s done'.

(SC1003)